SoftLayer/CLI/vlan/create.py

```diff
@@ -0,0 +1,54 @@
+"""Order/create a VLAN instance."""
+# :license: MIT, see LICENSE for more details.
+import click
+import SoftLayer
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@click.option('--name', required=False, prompt=True, help="Vlan name")
+@click.option('--datacenter', '-d', required=False, help="Datacenter shortname")
+@click.option('--pod', '-p', required=False, help="Pod name. E.g dal05.pod01")
+@click.option('--network', default='public', show_default=True, type=click.Choice(['public', 'private']),
+              help='Network vlan type')
+@click.option('--billing', default='hourly', show_default=True, type=click.Choice(['hourly', 'monthly']),
+              help="Billing rate")
+@environment.pass_env
+def cli(env, name, datacenter, pod, network, billing):
+    """Order/create a VLAN instance."""
+
+    item_package = ['PUBLIC_NETWORK_VLAN']
+    complex_type = 'SoftLayer_Container_Product_Order_Network_Vlan'
+    extras = {'name': name}
+    if pod and not datacenter:
```

Comment:
This should be just `if pod` I think. If a customer puts in both pod and datacenter, the pod will be ignored.
Ideally if a customer puts in pod and datacenter, datacenter should be ignored since it is less specific than pod. 

We don't need an error if a users puts both pod and dc, so no worries there. 
```suggestion
    if pod:
```